samples/gltf/src/main/java/com/google/ar/sceneform/samples/gltf/MainActivity.java

```diff
@@ -34,7 +36,7 @@
 public class MainActivity extends AppCompatActivity implements
         FragmentOnAttachListener,
         BaseArFragment.OnTapArPlaneListener,
-        ArFragment.OnViewCreatedListener {
+        ArFragment.OnViewCreatedListener, BaseArFragment.OnSessionConfigurationListener {
```

Comment:
This should be put on a new line together with `BaseArFragment.OnTapArPlaneListener`.

Reply:
done